relayer/cmd/run/beefy/command.go

```diff
@@ -94,24 +94,20 @@ func run(_ *cobra.Command, _ []string) error {
 			return err
 		}
 
-		err = relay.Start(ctx, eg)
```

Comment:
This changing boilerplate code which is shared by all our Go relayers. Its also unnecessary, since `relay.Start` spawns its own go routines internally.

Reply:
https://github.com/Snowfork/snowbridge/pull/1449/commits/b072474d551f6ccdce907c7e5cdc1cb69ff4b453